unit_tests/src/test_server_properties.cpp

```diff
@@ -29,3 +31,38 @@ TEST_CASE("server_properties", "[exceptions]")
                                  irods::KW_CFG_DB_PASSWORD));
     }
 }
+
+TEST_CASE("server_property_threadsafe", "[exceptions]")
+{
+    SECTION("Threads don't blow up")
+    {
```

Comment:
Consider consolidating these since there's only one test.

e.g. `TEST_CASE("server_properties is thread-safe", "[thread-safety]")`